monkey/infection_monkey/model/host.py

```diff
@@ -7,6 +7,7 @@
         self.domain_name = str(domain_name)
         self.os = {}
         self.services = {}
+        self.icmp = False
```

Comment:
Maybe this should be part of the services?

Reply:
I thought about it, but ICMP isn't really a service. Services might be the wrong name for that all together, but, like you said, it's a minor detail that we can revisit later.